index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Solitaire</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="css/common.css">
+  <script src="js/jquery-2.2.2.min.js" defer></script>
+  <script src="js/bootstrap.min.js" defer></script>
+
+  <script src="js/page.js" defer></script>
+  <script src="js/app.js" defer></script>
+</head>
+<body class="scroll-hide">
+<nav role="navigation" class="navbar navbar-default navbar-fixed-top">
+  <div class="container">
+    <div class="navbar-header">
+      <a class="navbar-brand">Solitaire</a>
+    </div>
+    <div id="navbarCollapse" class="collapse navbar-collapse">
+      <ul class="nav navbar-nav">
+        <li><a href="#" onclick="page.newGame(); this.blur()">New game</a></li>
+      </ul>
+    </div>
+  </div>
+</nav>
+<div class="container" data-component="page">
+    <div class="row seven-cols">
+      <div class="col-md-1"><div data-component="base"></div></div>
+      <div class="col-md-1"><div class="empty" data-component="base-2"></div></div>
+      <div class="col-md-1"></div>
+      <div class="col-md-1"><div class="empty" data-component="home-1" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div class="empty" data-component="home-2" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div class="empty" data-component="home-3" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div class="empty" data-component="home-4" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+    </div>
+    <div class="space"></div>
+    <div class="row seven-cols">
+      <div class="col-md-1"><div data-component="stack-1" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div data-component="stack-2" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div data-component="stack-3" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div data-component="stack-4" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div data-component="stack-5" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div data-component="stack-6" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
+      <div class="col-md-1"><div data-component="stack-7" ondrop="page.drop(event)" ondragover="page.allowDrop(event)"></div></div>
```

Comment:
> ondrop="page.drop(event)" ondragover="page.allowDrop(event)
Putting this on every div does not seem like a really good approach, in my humble opinion.